### PR TITLE
BoxPlot updates

### PR DESCRIPTION
**Pull request by kalletlak**
Fix https://github.com/cBioPortal/cbioportal/issues/6756
Fix https://github.com/cBioPortal/cbioportal/issues/6775

**Box plot tooltip**
Screen Shot 2019-11-05 at 12 47 38 PM

**Wider Box and whiskers than data points**
Screen Shot 2019-11-05 at 3 37 42 PM

### CODE REVIEW

File: src/pages/groupComparison/ClinicalData.tsx [hunk 1]
```diff
@@ -351,6 +351,46 @@ export default class ClinicalData extends React.Component<IClinicalDataProps, {}
         }
     }
 
+    @computed get boxPlotTooltip() {
+        return (d: any) => {
+            let content;
+            if (this.boxPlotData.isComplete) {
```
Comment (alisman) [inline src/pages/groupComparison/ClinicalData.tsx]: is it even possible for the tooltip to show even the boxplot data hasn't completed?

File: src/pages/groupComparison/ClinicalData.tsx [hunk 2]
```diff
@@ -351,6 +351,46 @@ export default class ClinicalData extends React.Component<IClinicalDataProps, {}
...
+                content = <table className="table table-bordered table-striped">
```
Comment (alisman) [inline src/pages/groupComparison/ClinicalData.tsx]: would rather not have the table-bordered unless super necessary.  just for consistency

File: src/pages/groupComparison/ClinicalData.tsx (hunk 1 above)
Comment (kalletlak) [reply src/pages/groupComparison/ClinicalData.tsx]: no

File: src/pages/groupComparison/ClinicalData.tsx (hunk 2 above)
Comment (kalletlak) [reply src/pages/groupComparison/ClinicalData.tsx]: done

**Comment by jjgao** [comment]
@kalletlak fyi - I updated the description above so that the issues can be automatically closed: https://help.github.com/en/github/managing-your-work-on-github/closing-issues-using-keywords#closing-multiple-issues

**Pull request merged**